tests/vmcheck/README.md

```diff
@@ -0,0 +1,30 @@
+Using vmcheck
+=============
+
+One time setup:
+
+```
+host$ vagrant up
+host$ vagrant ssh
+vm$ rpm-ostree rebase centos-atomic-continuous:centos-atomic-host/7/x86_64/devel/continuous
+vm$ systemctl reboot -r
+```
+
+Edit source code on the host system; to synchronize, use:
+
+```
+vagrant rsync
+```
+
+To build and install into the VM:
+
+```
+
+host$ vagrant ssh
+vm$ cd ~/src/tests/vmcheck
```

Comment:
Should this be `~/sync/tests/vmcheck`?
Or is the actual dir sync'ed to depend on a user configuration somewhere? (It's `sync/` on mine).